exp/addons/internal/controllers/suite_test.go

```diff
@@ -46,6 +47,19 @@ func TestMain(m *testing.M) {
 	}
 
 	setupReconcilers := func(ctx context.Context, mgr ctrl.Manager) {
+		// Create partial cache analog to main.go.
+		partialSecretCache, err := cache.New(mgr.GetConfig(), cache.Options{
+			Scheme:     mgr.GetScheme(),
+			Mapper:     mgr.GetRESTMapper(),
+			HTTPClient: mgr.GetHTTPClient(),
```

Comment:
Let's hard-code 10m resyncPeriod here (DefaultNamespaces is fine as the default is nil)